eo-runtime/src/main/java/org/eolang/PhTraced.java

```diff
@@ -58,7 +60,7 @@ public final class PhTraced implements Phi {
     /**
      * Locator of encaged object.
      */
-    private final Integer locator;
+    private final Integer locatr;
```

Comment:
@c71n93 what was wrong with `locator` name?

Reply:
@maxonfjvipon it repeats the method name (prohibited by PMD)